src/App.tsx

```diff
@@ -1,61 +1,137 @@
+/* eslint-disable no-console */
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState('0');
+  const [visibleTodos, setVisibleTodos] = useState(todosFromServer);
+  const [errorTitle, setErrorTitle] = useState(false);
+  const [errorUser, setErrorUser] = useState(false);
+
+  const handleTitle = (value: string) => {
+    setErrorTitle(false);
+    const re = /^[A-Za-z0-9а-яА-ЯёЁіІїЇ ]+$/;
+
+    if (re.test(value) || value === '') {
+      setTitle(value);
+    }
+  };
+
+  const newTodoId = () => {
```

Comment:
try to name function as verbs. Than it become clear it's a function. generateTodoId, for example